pontus_mapping/pontus_mapping/cluster_coord.py

```diff
@@ -135,15 +147,15 @@ def pointcloud_callback(self, msg: PointCloud2) -> None:
             transformed_msg, field_names=('x', 'y', 'z'),
             skip_nans=True
         )
-        points[:, 1] = -points[:, 1] #TODO ask why do this?
+        points[:, 1] = -points[:, 1]  # TODO ask why do this?
```

Comment:
Wait did we not remove this? I think this was only for making the sonar indices work with the occupancy grid. I think it might cause problems in this case